index.js

```diff
@@ -304,3 +305,5 @@ module.exports = function(configObject) {
     }
   }
 }
+
+module.exports.payload = payload
```

Comment:
I have created a library called iopipe-payload. This makes it easier to control the schema for Node and Python from a single repository and can be a place to test that schema (relatively) easily.